apps/condo/domains/notification/schema/helpers.js

```diff
@@ -0,0 +1,20 @@
+const faker = require('faker')
```

Comment:
it's the too-common name for utility package. And we don't put such utilities inside the `schema` folder

Reply:
moved to utils/testSchema